packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -1056,7 +1056,7 @@ class FlutterPlugin implements Plugin<Project> {
                             ? outputDirectory.get()
                             : outputDirectory
                         String filename = "app"
-                        String abi = output.getFilter(OutputFile.ABI)
+                        String abi = output.getFilter("ABI")
```

Comment:
Why stop using a constant and move to a string? that feels like a regression.

Reply:
I did this because for some unknown to me reason, the `com.android.build.OutputFile` import works only at compile time, but throws a ClassNotFound exception at runtime.

`OutputFile.ABI` is the only usage of this import, so I made a little hack and replaced it with the string "ABI" just to get it working.

Reply:
This is fixed, i.e. I went back to using a constant.